client/src/components/ProfileSidebar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Grid } from '@material-ui/core';
+import { Link } from 'react-router-dom';
+
+const ProfileSidebar = () => {
+    return (
+        // Sidebar container
+        <Grid container>
```

Comment:
Take a look at the Drawer component to make this sidebar: https://material-ui.com/components/drawers/